### ISSUE TITLE
[FTR] Hide caption for waterfall feed layout

### ISSUE THREAD

**Issue by Martin5001** [austinhuang0131/barinsta#1158]
I changed my settings to not show post captions by default to make things a bit more minimalistic, but this setting will only work for 2 or 3 column layouts.

I believe this is a bug and it's meant to apply to the single-column layout (Instagram default) as well, right? (If it's not a bug, please consider this change anyway to give users a choice to change this setting.)

Barinsta 19.2.1
Android 9.0
Samsung Galaxy J7

**Comment by austinhuang0131** [comment]
It's only meant for the single-post-viewing dialog (PostViewV2Fragment) so it's not a bug

**Comment by Martin5001** [comment]
Sorry, I should've made this more clear, but I actually meant the single post viewing dialog. The caption is shown there.

**Comment by austinhuang0131** [comment]
I meant the dialog that shows after you click a post

**Comment by Martin5001** [comment]
> I meant the dialog that shows after you click a post

Right, then it is a bug, because the caption shows there too (automatically, without me tapping any button) even though it's set to be hidden. Sorry again for making my issue not clear.

**Comment by austinhuang0131** [comment]
can't reproduce this

**Comment by Martin5001** [comment]
> can't reproduce this

Can I help in any way? Send some logs maybe?

**Comment by Martin5001** [comment]
So weirdly it now started working just fine... Don't know why, I didn't do anything...it might be that a restart helped. So I tried to test this and toggled the setting on and off few times with and without restart to see if it is that restart is actually needed to apply the setting, but it works just fine without restart...so I don't know. At least it seems to be working now. I'll keep you updated if I notice anything.

**Comment by austinhuang0131** [comment]
wait now i'm confused of my organization, whatever, i uh
